### PR TITLE
moving labs API iiif catalog to prod service

### PR DESCRIPTION
**Pull request by mekarpeles**
This PR has been hacked together as a proof of concept and not tested but should be useful in closing:
* #66

### CODE REVIEW

File: iiify/resolver.py [hunk 1]
```diff
@@ -29,14 +34,74 @@ def purify_domain(domain):
...
 
-def getids(q, limit=1000, cursor=''):
-    r = requests.get('%s/iiif' % apiurl, params={
-        'q': q,
-        'limit': limit,
-        'cursor': cursor
-    }, allow_redirects=True, timeout=None)
+def getids(q, limit=1000, cursor='', page=1):
+        q = request.args.get('q', '')
+        query = "(mediatype:(texts) OR mediatype:(image))" + \
```
Comment (saracarl) [inline iiify/resolver.py:39]: What about AV items?

**Comment by glenrobson** [comment]
The two times we use the getids method is:

```
@app.route('/iiif/')
def index():
    """Lists all available book and image items on Archive.org"""
    cursor = request.args.get('cursor', '')
    q = request.args.get('q', '')
    return jsonify(getids(q, cursor=cursor))

@app.route('/iiif/collection.json')
def catalog():
    cursor = request.args.get('cursor', '')
    q = request.args.get('q', '')
    domain = purify_domain(request.args.get('domain', request.url_root))
    return ldjsonify(collection(domain, getids(q, limit, cursor)['ids']))
```

**Comment by glenrobson** [comment]
Live URLs 500: 

https://iiif.archive.org/iiif/collection.json
https://iiif.archive.org/iiif/

**Comment by glenrobson** [comment]
Archivelab:

https://iiif.archivelab.org/iiif/

This doesn't work:
https://iiif.archivelab.org/iiif/collection.json

**Comment by glenrobson** [comment]
Need to point this to the spring-2024 branch and get the fix deployed for testing.

**Comment by mekarpeles** [comment]
https://iiif.archive.org/iiif/ and https://iiif.archivelab.org/iiif/ both work for me, this may have been an ephemeral issue.

https://iiif.archivelab.org/iiif/collection.json definitely seems broken and needs investigation

**Pull request merged**